blocks/v2-video/v2-video.css

```diff
@@ -0,0 +1,117 @@
+/* stylelint-disable selector-class-pattern */
+.v2-video-wrapper {
+  padding: 0 !important;
+  color: var(--c-white);
+  margin: 0 !important;
+  width: 100% !important;
+}
+
+.v2-video {
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  clip-path: inset(24px 24px round 8px);
+  transition: clip-path 0.3s ease-in-out;
+}
+
+.v2-video.v2-video--full-width {
```

Comment:
I was told by @cogniSyb to just use kebab case in class names. Here I see plenty of different types:
- v2-video--full-width
- v2-video__video
Can we align on just using something in particular? 

cc @Lakshmishri @cogniSyb

Reply:
The `v2-video` is a block `full-width` is a BEM modifier so the full class name is `v2-video--full-width` (`--` before the modifier).
The `video` is an element inside the `v2-video` so the class name is `v2-vidoe__video` (`__` before the block)